packages/adapters-library/src/core/utils/unwrap.ts

```diff
@@ -87,7 +108,8 @@ async function fetchUnwrapExchangeRates(
       !(
         error instanceof
         ProtocolSmartContractNotDeployedAtRequestedBlockNumberError
-      )
+      ) &&
+      !(error instanceof NotImplementedError)
```

Comment:
I had to add this because with convex, it returns a protocol token with no underlyings other than claimable rewards. With the new logic that meant it would be unwrapped, but that was causing an error due to the convex curve wrapper adapter not implementing unwrap.

Reply:
ah i see...perhaps we need to implement unwrap, one for another day